searchPage/search.js

```diff
@@ -1,5 +1,265 @@
+"use strict";
 var menu = document.getElementById("display-menu");
+const searchCountry = document.getElementById("countries"); // the datalist of countires
+const inputCountry = document.getElementById("input"); // the choosen country
+let selectedCountry; // storing the choosen country
+//listing all countries
+const countries = [
```

Comment:
check the ability to use an API  to get all countries rather than adding them manually.

Reply:
We had to render all the countries locally without fetching, because of two reasons. First, when we fetch for all the countries in the API sometimes it falls and might not give us the wanted data. Second, the two APIs that we use have different approaches to search for certain countries, and that makes conflict.